packages/rum/test/node/index-export.spec.js

```diff
@@ -41,8 +41,10 @@ describe('apm base', () => {
     /**
      * Delete module cache and run bootstrap again
      */
-    delete require.cache[require.resolve('@elastic/apm-rum')]
-    delete require.cache[require.resolve('@elastic/apm-rum/dist/lib/bootstrap')]
+    let cache = require.cache
```

Comment:
you could just do delete require.cache or set it to empty object instead of looping over

Reply:
@vigneshshanmugam , setting require.cache won't work, since it seems the same object ref is kept somewhere else so we have to delete its keys.

Reply:
I am removing this test in my next PR #796 so lets not spend more time. I am good with it.